bin/flutterflow_cli.dart

```diff
@@ -130,6 +131,11 @@ ArgResults _parseArgs(List<String> args) {
       help: 'Generate the project with debug logging to be able to use '
           'FlutterFlow Debug Panel inside the DevTools.',
       defaultsTo: false,
+    )
+    ..addOption(
+      'environment-name',
```

Comment:
We already have flag called "environment" (line 39)... Consider renaming this to something that would disambiguate these flags, e.g. `--project-environment` or `--dev-environment`.

Up to you.

Reply:
Good call!